diff --git a/dist/esm/notification.mjs b/dist/esm/notification.mjs
--- a/dist/esm/notification.mjs
+++ b/dist/esm/notification.mjs
@@ -18 +18 @@
-import jsonLineParser from 'stream-json/jsonl/Parser.js';
+import { parser as jsonLineParser } from 'stream-json/jsonl/parser.js';
@@ -176,7 +176,7 @@ export class NotificationPoller extends EventEmitter {
       bucketName: this.bucketName,
       query
     }, '', [200], region).then(response => {
-      const asm = jsonLineParser.make();
+      const asm = jsonLineParser.asStream();
       pipesetup(response, asm).on('data', data => {
         // Data is flushed periodically (every 5 seconds), so we should
         // handle it after flushing from the JSON parser.
diff --git a/dist/main/notification.js b/dist/main/notification.js
--- a/dist/main/notification.js
+++ b/dist/main/notification.js
@@ -7 +7 @@
-var _Parser = require("stream-json/jsonl/Parser.js");
+var _Parser = require("stream-json/jsonl/parser.js");
@@ -196,7 +196,7 @@ class NotificationPoller extends _eventemitter.EventEmitter {
       bucketName: this.bucketName,
       query
     }, '', [200], region).then(response => {
-      const asm = _Parser.make();
+      const asm = _Parser.parser.asStream();
       (0, _helper.pipesetup)(response, asm).on('data', data => {
         // Data is flushed periodically (every 5 seconds), so we should
         // handle it after flushing from the JSON parser.
diff --git a/src/notification.ts b/src/notification.ts
--- a/src/notification.ts
+++ b/src/notification.ts
@@ -18 +18 @@
-import jsonLineParser from 'stream-json/jsonl/Parser.js'
+import { parser as jsonLineParser } from 'stream-json/jsonl/parser.js'
@@ -216,7 +216,7 @@ export class NotificationPoller extends EventEmitter<{
 
     this.client.makeRequestAsync({ method, bucketName: this.bucketName, query }, '', [200], region).then(
       (response) => {
-        const asm = jsonLineParser.make()
+        const asm = jsonLineParser.asStream()
 
         pipesetup(response, asm)
           .on('data', (data) => {
